lib/menu.rb

```diff
@@ -0,0 +1,44 @@
+
+class Menu
+
+  DISHES = [
+    { id: 1, name: 'Spinach & Cheddar Pancakes', price: 7, available?: true, quantity: 0 },
+    { id: 2, name: 'Veggie Breakfast', price: 8, available?: true, quantity: 0 },
+    { id: 3, name: 'Halloumi Sandwhich', price: 7, available?: true, quantity: 0 },
+    { id: 4, name: 'Soup of the Day', price: 8, available?: true, quantity: 0 },
+    { id: 5, name: 'Ceaser Salad', price: 8, available?: true, quantity: 0 },
+    { id: 6, name: 'Pizza', price: 9, available?: true, quantity: 0 },
+    { id: 7, name: 'Vegan Pizza', price: 10, available?: true, quantity: 0 },
+    { id: 8, name: 'Yum Yum Salad', price: 11, available?: true, quantity: 0 },
+    { id: 9, name: 'Singapore Laksa', price: 11, available?: true, quantity: 0 },
+    { id: 10, name: 'Spiced Noodles', price: 11, available?: true, quantity: 0 },
+    { id: 11, name: 'Phad Thai', price: 11, available?: true, quantity: 0 },
+    { id: 12, name: 'Caponata', price: 11, available?: false, quantity: 0 }
+  ].freeze
+
```

Comment:
There are some complexities in using a structure like this, when it comes to traversing each data point later in your implementation to print / change dishes. 

I'll share an example I wrote for another dev: 

Hashes are most useful where you have a key value pair - eg. if the only information you needed to record was the item name and price, a simple hash could look like:

```
@menu = { "1. Mattar Paneer" => 12.50, "Black Daal" => 7.50 }

@menu[item_name] will return the cost of that item.
```

For data requiring more than two values, where key/value isn't enough, I would suggest defining a new object, for example:

```
class Dish
    def initialize(name, price)
        @name = name
        @price = price
    end

    # getter methods for retrieving information
end
```

This way, you could work with an array of Dishes inside your Menu class, eg. @menu = [Dish.new(args), Dish.new(args), etc.]

Singular dish data can be accessed with @menu[0].name, etc. Which can be quite a bit easier than traversing a complex hash!